### PR TITLE
Add unsafe routes to swagger

### LINKED ISSUE #4047: docs: `/dial_peers` is missing from `rpc/swagger.yaml`
I can't find doc on `/dial_peers` endpoint in https://tendermint.com/rpc/

### PR DESCRIPTION
**Pull request by marbar3778**
- added `/dial_peers` & `dial_seeds` to swagger docs under unsafe tag

closes #4047

Signed-off-by: Marko Baricevic <marbar3778@yahoo.com>

* [ ] Referenced an issue explaining the need for the change
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG_PENDING.md

### CODE REVIEW

File: rpc/swagger/swagger.yaml [hunk 1]
```diff
@@ -2675,3 +2737,22 @@ definitions:
...
+    type: object
+    properties:
+      Persistent:
+        type: bool
+        example: false
+      Peers:
+        type: Array
+        items:
+          type: "string"
+        example: ["peer:id"]
```
Comment (melekes) [inline rpc/swagger/swagger.yaml]:
```suggestion
        example: ["deadbeefdeadbeefdeadbeefdeadbeefdeadbeef@192.168.1.2:26656"]
```

File: rpc/swagger/swagger.yaml [hunk 2]
```diff
@@ -2675,3 +2737,22 @@ definitions:
...
+  dialSeedsPost:
+    type: object
+    properties:
+      Peers:
+        type: Array
+        items:
+          type: "string"
+        example: ["peer:id"]
```
Comment (melekes) [inline rpc/swagger/swagger.yaml]:
```suggestion
        example: ["deadbeefdeadbeefdeadbeefdeadbeefdeadbeef@192.168.1.2:26656"]
```

File: rpc/swagger/swagger.yaml [hunk 3]
```diff
@@ -409,6 +409,68 @@ paths:
...
+      summary: Add Peers/Persistent Peers (unsafe)
+      operationId: dial_peers
+      tags:
+        - unsafe
+      description: |
+        Set a persistent peer, this route in under unsafe, and has to manually enabled to use
+      consumes:
+        - application/json
+      parameters:
+        - description: string of possible peers, can be added as persistent peers
```
Comment (melekes) [inline rpc/swagger/swagger.yaml]: there's also `persistent` bool argument!

File: rpc/swagger/swagger.yaml [hunk 4]
```diff
@@ -451,7 +451,7 @@ paths:
       consumes:
         - application/json
       parameters:
-        - description: string of possible peers, can be added as persistent peers
+        - description: string of possible peers, bool arguement if they should be added as persistent
```
Comment (melekes) [inline rpc/swagger/swagger.yaml]: typo in arguement

File: rpc/swagger/swagger.yaml [hunk 5]
```diff
@@ -2747,12 +2747,12 @@ definitions:
         type: Array
         items:
           type: "string"
-        example: ["peer:id"]
+        example: ["deadbeefdeadbeef@192.168.1.2:26656"]
```
Comment (melekes) [inline rpc/swagger/swagger.yaml]: let's use real ID, e.g. 6f172048b821e3b1ab98ffb0973ba737966eecf8

File: rpc/swagger/swagger.yaml [hunk 6]
```diff
@@ -2747,12 +2747,12 @@ definitions:
...
   dialSeedsPost:
     type: object
     properties:
       Peers:
         type: Array
         items:
           type: "string"
-        example: ["peer:id"]
+        example: ["deadbeefdeadbeefdeadbeef@192.168.1.2:26656"]
```
Comment (melekes) [inline rpc/swagger/swagger.yaml]: same 6f172048b821e3b1ab98ffb0973ba737966eecf8

File: rpc/swagger/swagger.yaml (hunk 5 above)
Comment (marbar3778) [reply rpc/swagger/swagger.yaml]:
```suggestion
        example: ["6f172048b821e3b1ab98ffb0973ba737966eecf8@192.168.1.2:26656"]
```

File: rpc/swagger/swagger.yaml (hunk 6 above)
Comment (marbar3778) [reply rpc/swagger/swagger.yaml]:
```suggestion
        example: ["6f172048b821e3b1ab98ffb0973ba737966eecf8@192.168.1.2:26656"]
```

File: rpc/swagger/swagger.yaml (hunk 4 above)
Comment (marbar3778) [reply rpc/swagger/swagger.yaml]:
```suggestion
        - description: string of possible peers, bool argument if they should be added as persistent
```

**Pull request merged**